common/network-common/src/main/java/org/apache/spark/network/util/TransportConf.java

```diff
@@ -257,4 +257,31 @@ public Properties cryptoConf() {
     return CryptoUtils.toCryptoConf("spark.network.crypto.config.", conf.getAll());
   }
 
+  /**
+   * When memory usage of Netty is above this water mark, it's regarded as memory shortage.
+   */
+  public Long nettyMemWaterMark() {
+    return JavaUtils.byteStringAsBytes(conf.get("spark.network.netty.memCostWaterMark", "1g"));
```

Comment:
Yes, I do think it's good to put the config into `org.apache.spark.internal.config`.
But I found it hard. Since `org.apache.spark.internal.config` in in `core` module. I didn't find a good way to import it from module `spark-network-yarn`. Did I miss something ?